### ISSUE TITLE
shap.plots.force issue

### ISSUE THREAD

**Issue by Hamid-Nasiri** [shap/shap#1908]
I can't seem to get shap.plots.force to work for the second plot on the readme (# visualize all the training set predictions)
This is the code I'm using and the error. Any help would be great.

Code: 

import xgboost
import shap
X, y = shap.datasets.boston()
model = xgboost.XGBRegressor().fit(X, y)
explainer = shap.Explainer(model)
shap_values = explainer(X)
shap.plots.force(explainer.expected_value, shap_values)

---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-13-89eab621d022> in <module>()
     10 explainer = shap.Explainer(model)
     11 shap_values = explainer(X)
---> 12 shap.plots.force(explainer.expected_value, shap_values)

c:\users\hamid\appdata\local\programs\python\python37\lib\site-packages\shap\plots\_force.py in force(base_value, shap_values, features, feature_names, out_names, link, plot_cmap, matplotlib, show, figsize, ordering_keys, ordering_keys_time_format, text_rotation, contribution_threshold)
    101 
    102     if type(shap_values) != np.ndarray:
--> 103         return visualize(shap_values)
    104 
    105     # convert from a DataFrame or other types

c:\users\hamid\appdata\local\programs\python\python37\lib\site-packages\shap\plots\_force.py in visualize(e, plot_cmap, matplotlib, figsize, show, ordering_keys, ordering_keys_time_format, text_rotation, min_perc)
    343             return AdditiveForceArrayVisualizer(e, plot_cmap=plot_cmap, ordering_keys=ordering_keys, ordering_keys_time_format=ordering_keys_time_format)
    344     else:
--> 345         assert False, "visualize() can only display Explanation objects (or arrays of them)!"
    346 
    347 class BaseVisualizer:

AssertionError: visualize() can only display Explanation objects (or arrays of them)!

**Comment by covix** [comment]
Hi @hamidnasiri1369, I have the same issue but https://github.com/slundberg/shap/issues/1883 seems to find a way to solve it.
Anyway I agree the README example should be updated!

**Comment by panshuaiyin** [comment]
So, have you solve the problem?@covix@hamidnasiri1369

**Comment by covix** [comment]
Yes, by calling `shap.plots.force(explainer.expected_value, shap_values.values)`

**Comment by mouzui** [comment]
shap.initjs() 
shap.plots.force(explainer.expected_value, shap_values.values[0], feature_names=feature_names1)
